File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/ViewPagerActivitySubcomponent.java
package ar.com.wolox.android.example.ui.viewpager;

import ar.com.wolox.android.example.ui.viewpager.fragment.ViewPagerFragmentModule;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;

@Subcomponent(modules = {ViewPagerFragmentModule.class})
public interface ViewPagerActivitySubcomponent extends AndroidInjector<ViewpagerActivity> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<ViewpagerActivity> {
    }
}

File: app/src/main/java/ar/com/wolox/android/example/ui/viewpager/ViewPagerActivitySubcomponent.kt
package ar.com.wolox.android.example.ui.viewpager

import dagger.Subcomponent
import dagger.android.AndroidInjector

@Subcomponent
interface ViewPagerActivitySubcomponent : AndroidInjector<ViewPagerActivity> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<ViewPagerActivity>()
}
